README.md

```diff
@@ -85,23 +80,22 @@ You can now run the executable in your app directory:
 bundle exec broadside --help
 ```
 
-`bundler` can also install binstubs for you - small scripts in the `/bin` directory of your application that will mean you can type `bin/broadside do_something` instead of `bundle exec broadside do_something`.  If you want to save the typing, run:
-
+You may also generate binstubs using
```

Comment:
i don't know if "generate binstubs" means anything to people who don't know ruby, and knowing ruby isn't really a pre-requisite for using broadside.